test/Microsoft.DotNet.Kusto.Tests/KustoClientProviderTests.cs

```diff
@@ -37,8 +37,8 @@ public async Task NoParameterQueryIsPassedPlainly()
                 q.ExecuteQueryAsync(Capture.In(dbNames), Capture.In(queries), Capture.In(properties), It.IsAny<CancellationToken>()))
             .Returns(Task.FromResult(reader));
             
-        using (var client = new KustoClientProvider(MockOptionMonitor.Create(new KustoClientProviderOptions
-                       {QueryConnectionString = "IGNORED-CONNECTION-STRING", Database = "TEST-DATABASE",}),
+        using (var client = new KustoClientProvider(MockOptionMonitor.Create(new KustoOptions
+                       {KustoClusterUri = "IGNORED-CONNECTION-STRING", Database = "TEST-DATABASE",}),
                    queryProvider.Object))
```

Comment:
nit: these blocks are difficult to read. I suggest
```suggestion
        var options = new KustoOptions
        {
            KustoClusterUri = "IGNORED-CONNECTION-STRING",
            Database = "TEST-DATABASE",
        };
        using (var client = new KustoClientProvider(
            MockOptionMonitor.Create(options),
            queryProvider.Object))
```

The options never seem to change in this class; could make them a `static readonly` field.